lms/static/scripts/frontend_apps/components/dashboard/AssignmentActivity.tsx

```diff
@@ -31,12 +27,24 @@ export default function AssignmentActivity() {
 
   return (
     <Card>
-      <CardHeader fullWidth>
-        <CardTitle tagName="h2" data-testid="title">
+      <CardHeader fullWidth classes="flex-col !gap-x-0 !items-start">
```

Comment:
What are the values being overridden by `!important` classes?

Reply:
`CardHeader` includes `gap-x-2` and `items-center`.

Reply:
Can you add a comment to explain this.